index.html

```diff
@@ -237,6 +236,11 @@ <h2>Attributes</h2>
                     and <code><a href="https://w3c.github.io/webrtc-stats/#dom-rtcinboundrtpstreamstats-removedsamplesforacceleration">removedSamplesForAcceleration</a></code>.
                     For video, this may result in the same frame being rendered
                     multiple times or frames may be dropped.
+
+                    With constant audio/video feed received under the perfect
+                    network conditions, for e.g. loopback connection over
+                    localhost, it SHOULD take less than 30 seconds on average to
+                    reach 1 second buffering level from empty buffer.
```

Comment:
Do we really need the "perfect network conditions" part? In a real-world use case you wouldn't have perfect conditions, so if the spec only mentions the perfect case then it's not as clear how this is to be used by real applications.

Can this say "The acceleration or deceleration rate may vary depending on network conditions or the type of audio received (e.g. speech or background noise). It MAY take several seconds to achieve 1 second of buffering but SHOULD not take more than 30 seconds assuming packets are being received. The speed MAY be different for audio and video."

Also can you move this to the previous paragraph (outside the "note")?

Reply:
Done.